src/lib.rs

```diff
@@ -266,31 +503,42 @@ impl<E> Error<E> {
     /// where the original error can transform into that type.
     #[inline]
     pub fn into_error<N>(self) -> N
-    where
-        E: Into<N>,
+        where
```

Comment:
If you feel confident enough with git, leaving out these unrelated formatting changes would be good.

Otherwise I'll just reformat with `rustfmt` before merging to fix these.

Reply:
I ran `rustfmt`, my self

Reply:
it's all good then! I think we must have different versions/configurations- I can run the version I have installed (`0.3.8-nightly (346238f 2018-02-04)`) after merging.